coding-guidelines.md

```
@@ -0,0 +1,344 @@
+# Avoid catching all exceptions
+try:
+    asset = self._assets[asset_id]
+except Exception:  # Too broad
+    raise AssetNotFoundError(f"Asset '{asset_id}' not found")
+```
+
+## Testing Standards
+
+### Test Organization
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)